src/components/EditableToDo.vue

```diff
@@ -0,0 +1,235 @@
+          and you will not be able to undo this action!
+        </p>
+        <div class="flex flex-row space-x-6 mb-2 font-semibold">
+          <ModalButton
+            value="Yes"
+            additional-classes="bg-green"
+            :modalButton="true"
+            @click="removeTodo"
+          />
+          <ModalButton
+            value="No"
+            additional-classes="bg-red"
+            :modalButton="true"
+            @click="hidePopUp"
+          />
+        </div>
+      </div>
+    </div>
+    <div class="flex flex-col m-[18px]">
+      <div class="flex flex-row justify-between items-center">
+        <input
+          v-model="currentTitle"
+          type="text"
+          placeholder="Title"
+          class="w-[197px] font-semibold text-lg lg:text-4xl"
+          :class="isDropDownShown ? 'opacity-30' : ''"
+          :disabled="isDropDownShown"
+        />
+        <div class="flex flex-row space-x-[5px] items- lg:hidden items-center">
+          <div
+            v-for="(priority, ind) in priorityList"
+            class="p-1 rounded-full h-fit"
+            :class="(getColorOfPriority(priority), styleOfPriorityButton(ind))"
+            @click="changePriority(priority)"
+          ></div>
+        </div>
+        <div class="hidden lg:flex flex-col" :class="positionOfDropDownButton">
+          <div
+            class="text-lg text-white font-semibold rounded-full"
+            :class="dropDownStyle"
+          >
+            <button
+              id="dropdownButton"
+              class="ml-5 mr-3 text-center flex items-center"
+              type="button"
+              :disabled="isPopUpShown"
+              @click="changeDropDownShown"
+            >
+              {{ currentPriority }}
+              <img class="pl-2" src="../assets/icons/dropdown-vector.svg" />
+            </button>
+          </div>
+          <!-- Dropdown menu -->
+          <div
+            v-if="isDropDownShown"
+            id="dropdown"
+            class="cursor-pointer absolute z-10 top-12 text-lg font-semibold rounded-xl border-black border-2 text-start items-center bg-white"
+          >
+            <ul class="py-2 pl-4 pr-3" aria-labelledby="dropdownButton">
+              <li v-for="priority in priorityList">
+                <p class="opacity-100" @click="changePriority(priority)">
+                  {{ priority }}
+                </p>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+      <div class="flex flex-row space-x-1" :class="currentOpacity">
+        <img src="../assets/icons/Vectorcalendar.svg" />
+        <p class="text-xs">{{ formatShortDate(date) }}</p>
+      </div>
+      <div class="self-start mt-4 mb-4">
+        <textarea
+          v-model="currentText"
+          placeholder="Lorem ipsum"
+          class="border-none h-[70.27px] w-[224.45px] lg:h-[103px] lg:w-[455.1px] text-start text-sm lg:text-lg text-grey"
```

Comment:
I dont think we should have classes like h-[70.27px] or w-[224.45px]

Reply:
Can you explain why you think that, just so he understands what you want in these cases and what is the cause for this concern?